src/components/tabs/js/tabsController.js

```diff
@@ -219,17 +310,24 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
    */
   function handleSelectedIndexChange (newValue, oldValue) {
     if (newValue === oldValue) return;
-    
+
     ctrl.selectedIndex     = getNearestSafeIndex(newValue);
     ctrl.lastSelectedIndex = oldValue;
     ctrl.updateInkBarStyles();
     updateHeightFromContent();
     adjustOffset(newValue);
+    if (ctrl.shrinkToolbar) {
+      bindScrollListener(newValue, oldValue);
+    }
     $scope.$broadcast('$mdTabsChanged');
     ctrl.tabs[ oldValue ] && ctrl.tabs[ oldValue ].scope.deselect();
     ctrl.tabs[ newValue ] && ctrl.tabs[ newValue ].scope.select();
   }
 
+  function scrollTop (index) {
+    document.querySelector('#tab-content-' + ctrl.tabs[index].id).scrollTop = 0;
```

Comment:
This is the same as `document.getElementById('tab-content-' + ctrl.tabs[index].id).scrollTop = 0;`.